magda-migrator-content-db/sql/V1_8__footer_default_config.sql

```diff
@@ -0,0 +1,43 @@
+INSERT INTO "public"."content" ("id", "type", "content") VALUES ('footer/navigation/categories/medium/data.gov.au', 'application/json', '{
+  "label": "Data.gov.au",
+  "links": [
+    [
+      "About",
+      "page/about"
+    ],
+    [
+      "Suggest a dataset",
+      "suggest"
+    ],
+    [
+      "Privacy Policy",
+      "page/privacy-policy"
+    ]
+  ]
+}'),
+('footer/navigation/categories/medium/developers', 'application/json', '{
+  "label": "Developers",
```

Comment:
This is not valid as far as content schema specified goes. E.g. order is missing.

Reply:
Have added order fields to all items